ajax/libs/rye/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "rye",
+  "filename": "rye.min.js",
+  "version": "0.0.1",
```

Comment:
the latest version is 0.1.0, you can add it by git log tracing, and ask the authors if they can keep adding git tags.